src/main/java/dev/emortal/velocity/resourcepack/ResourcePackForcer.java

```diff
@@ -64,14 +65,15 @@ private void updateResourcePackInfo(@NotNull ProxyServer proxy) {
     @SneakyThrows
     private byte[] fetchSha1() {
         MessageDigest digest = MessageDigest.getInstance("SHA-1");
-        InputStream fileInputStream = new URL(PACK_URL).openStream();
+        InputStream fileInputStream = new URI(PACK_URL).toURL().openStream();
```

Comment:
What's the difference here? Why do we need to create a URI then convert when we can just create a URL in the first place?

Reply:
The URL constructor is deprecated now, and the recommended replacement is to create a URI and convert it to a URL. I think they're trying to slowly move everyone away from URLs, as the URL API is very old and is not as good as the URI API.